README.md

```diff
@@ -1,52 +1,65 @@
 # service.routing [![Build Status](https://travis-ci.org/hiposfer/service.routing.svg?branch=master)](https://travis-ci.org/hiposfer/service.routing)
 
-FIXME: description
+A routing engine service using Open Street Map as data source. Its current focus are
+flexibility, speed and stability in that order.
 
-## Installation
-
-Download from http://example.com/FIXME.
-
-### With Docker
+## Usage
 
-You can always grab the latest docker image and run it directly:
+`service.routing` creates an API server with a Swagger interface for your convenience.
+You can find it at [localhost:3000](http://localhost:3000). For a quick test, you can
+check [try.hyposfer.com](http://try.hiposfer.com/index.html)
 
-    $ docker run -it -p 3000:3000 hiposfer/service.routing
+### Production
 
-You can also build a docker image locally out of your changes:
+- locally
 
-    $ docker build -t somename .
 
-Or using docker-compose within the source directory:
+    $ lein uberjar
+    $ java -jar target/routing.jar
 
-    $ docker-compose up
 
-Then open the Swagger API documentation at [localhost:3000](http://localhost:3000).
+- With Docker
 
-## Usage
+Grab the latest docker image and run it:
 
-FIXME: explanation
+    $ docker run -it -p 3000:3000 hiposfer/service.routing
 
-[//]: # ($ java -jar sagacious-woof-0.1.0-standalone.jar [args])
 
+### Development
 
-## Options
+We follow [Stuart Sierra's reload workflow](https://github.com/stuartsierra/component).
+The system configuration is created [here](https://github.com/hiposfer/service.routing/blob/master/src/hiposfer/service/routing/core.clj#L77)
+by merging the [default config](https://github.com/hiposfer/service.routing/blob/master/resources/default-config.edn)
+with the [required environmental variables](https://github.com/hiposfer/service.routing/blob/master/src/hiposfer/service/routing/core.clj#L84).
```

Comment:
We better avoid pointing to source code lines in README. Who is going to take care of updating these links after all!? :)

Reply:
Good point, I will change this.